src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    public String color;
+    public int number;
```

Comment:
Fields 'color' and 'number' should have access modifiers to encapsulate the data. Consider using 'private' to restrict direct access and provide getters and setters if necessary.